pandas/tests/io/excel/test_style.py

```diff
@@ -110,12 +119,31 @@ def test_styler_to_excel_unstyled(engine):
     ),
 ]
 
+# List of (engine, css) combinations which are expected to fail
+xfail_params: List[Tuple[str, str]] = []
+
+
+@pytest.fixture
+def xfail_selected_styles(request):
+    """Check to see if specific engine-parameter combinations are expected to fail"""
+    css = request.getfixturevalue("css")
+    expected = request.getfixturevalue("expected")
+    engine = request.getfixturevalue("engine")
+
+    if isinstance(expected, dict):
+        expected = expected[engine]
+
+    if (engine, css) in xfail_params:
```

Comment:
Hmm not sure if I completely understand the testing setup, but since `xfail_params` is empty above and not filled otherwise, isn't this case always false?

Reply:
Correct, I had to add an intermediate modification (`xfail_params = [("xlsxwriter","vertical-align: middle;")]`) so that it would correctly xfail before I added the bug fix. Once it was fixed, I removed that entry since it's no longer an xfail.

I could remove the `xfail_params` section completely now, but it could be useful in the future if there are other non-implemented styles. Let me know what you prefer.

Reply:
I think it should be removed for now just so it's clear to all developers what's immediately utilized code.